src/udev.c

```diff
@@ -28,6 +28,17 @@
 
 #include "project.h"
 
+#define COMMUNICATIONS_CLASS            0x02
+#define WIRELESS_CLASS                  0xE0
+#define VENDOR_SPECIFIC_CLASS           0xFF
+#define ETHERNET_NETWORKING_SUBCLASS    0x06
+#define RADIO_FREQUENCY_SUBCLASS        0x01
+#define BLUETOOTH_PROTOCOL              0x01
```

Comment:
classes.h is used to translate the 1-byte class/interface/protocol names to their string representations, which isn't really what is needed (though it could technically be used for our purpose, it would be inefficient and more complicated). I moved the defines to classes.h.